### PR TITLE
Fix for v2 release

### PR DESCRIPTION
**Pull request by nikolay-turpitko**
Hey! According to this document: https://github.com/golang/go/wiki/Modules#releasing-modules-v2-or-higher, you should add `.../v2` part to your module path in the `go.mod`. I tried to import your version and it rejected v2. I tried to fix it on my branch and it seems to work. Your branch seems most up-to-date so far. Original project author seems inactive. Have you tried to send a PR to him? Thx.

### CODE REVIEW

**Comment by nikolay-turpitko** [comment]
Also, notice that request/response bodies were not logged in my tests, I've fixed that by moving call to the next handler to the point after req/res are modified. I suppose, they didn't used.

File: go.mod [hunk 1]
```diff
@@ -1,7 +1,9 @@
-module github.com/bakatz/echo-logrusmiddleware
+module github.com/nikolay-turpitko/echo-logrusmiddleware/v2
```
Comment (bakatz) [inline go.mod]: please commit this as `bakatz` instead of your username :)

Comment (nikolay-turpitko) [reply go.mod]: Fixed

**Review by bakatz — APPROVED** [review]

**Comment by bakatz** [comment]
@nikolay-turpitko thanks for the fixes, merged - the original project author is inactive and doesn't accept pull requests.

**Pull request merged**